src/content/self-hosted/administration/configuration.mdx

```diff
@@ -975,9 +975,13 @@ userDefinedNamespaces: false
 
 When enabled Okteto automatically adds a preferred affinity to every pod, which leans towards placing pods from the same namespace and in the same node (default: `true`).
 
+ - `antiaffinity`: When enabled Okteto adds a pod anti-affinity to ensure all pods in same Namespace are placed together. Disabled by default
+
 ```yaml
 userPodAffinity:
   enabled: true
+  antiaffinity: 
```

Comment:
@pchico83 can we fix the casing to `antiAffinity` so it's consistent with the rest of the properties?